frontend/src/components/Button.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'; // React
+import PropTypes from 'prop-types'; // PropTypes
+
+import { ReactComponent as Edit } from 'assets/edit.svg'; // Edit Icon
+import styles from 'styles/components/Button.module.css'; // Component styles
+
+export default function Button({ className = '', children, variant, ...props }) {
+  switch (variant) {
+    case 'edit':
+      return (
+        <button className={`${styles.button__edit} ${className}`} {...props}>
+          <Edit />
+        </button>
+      );
+    case 'outlined':
+      return (
+        <button className={`${styles.button} ${styles.button__outlined} ${className}`} {...props}>
+          {children}
+        </button>
+      );
+    default:
```

Comment:
Might be a small thing - since the variant is set to oneOf ['outlined', 'contained', 'edit'] maybe 'contained' can be it's own case? Then default could just be a standard button. This might help in the future if say we add more types of buttons. Again, very small nitpick

Reply:
I think it's good to keep the `contained` variant as the default case (similar to MUI) since it is already styled, as opposed to a default HTML button. If we want to add more button styles in the future, we can easily just add more cases to this switch.